test/pylib/scylla_cluster.py

```diff
@@ -733,7 +733,7 @@ def before_test(self, name) -> None:
         if self.start_exception:
             # Mark as dirty so further test cases don't try to reuse this cluster.
             self.is_dirty = True
-            raise self.start_exception
+            raise Exception(f'Exception when starting cluster {self}:\n{self.start_exception}')
```

Comment:
In theory we could try to inspect the start exception. Maybe use exception chaining?